esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/workflow.rst

```diff
@@ -72,6 +73,7 @@ and ``opt`` directories within the workflow. The files required are:
 
 ``opt/rose-suite-<site>.conf``
   Contains configuration items specific to the ``SITE``, including ``SITE``
+  Defines paths specific to the ``SITE`` on where to find required datasets
```

Comment:
This sentence duplicates what is in the previous sentence; would it be possible to reword this to be a continuation of the previous sentence, please?

Reply:
e8c73489a5ad51ebf08d5dd3ee5ffbe802663522